src/aat/java/uk/gov/hmcts/ccd/fta/steps/BackEndFunctionalTestScenarioHooks.java

```diff
@@ -0,0 +1,18 @@
+package uk.gov.hmcts.ccd.fta.steps;
```

Comment:
Do we need this class? (maybe it gets used somewhere outside this pr?) looks like this class could be removed or perhaps be used in the BackEndFunctionalTestScenarioPlayer.java class in order to get access to the `Scenario` object that way rather than another @before step

Reply:
will push in the next commit